packages/i18n/src/index.js

```diff
@@ -150,7 +150,7 @@ export function _nx( single, plural, number, context, domain ) {
  * original format string is returned.
  *
  * @param {string}   format  The format of the string to generate.
- * @param {string[]} ...args Arguments to apply to the format.
+ * @param {string[]} args Arguments to apply to the format.
```

Comment:
This is wrong though. It would imply I call the function as `sprintf( '%s %d', [ 'foo', 10 ] );` when in fact it should be `sprintf( '%s %d', 'foo', 10 );`, which is how the spread documentation is meant to indicate.

Related: https://github.com/jsdoc3/jsdoc/issues/1074

Reply:
Ah, it's a variadic argument. So, we should use this instead:

```suggestion
 * @param {...string} Arguments to apply to the format.
```

It's not the variable who should document this, but the type, as per [`@type`](http://usejsdoc.org/tags-type.html).

Reply:
Fixed at fa197c5

Reply:
>Ah, it's a variadic argument. So, we should use this instead:

Yep! Good call. Feels like we should lint against this, since I assume the previous format was just wildly invalid?